### PR TITLE
Decimal places

### PR DESCRIPTION
**Pull request by thecristen**
Exporting and analyzing checkin data was made cumbersome by the extreme precision of our calculations, which sometimes indicated infinitesimally small differences that we ought dismiss as insignificant. This PR adds some changes that will keep our sig figs to a manageable number (3).

This is also my first foray into tests! I added two tests for the changes in this PR, as well as added a fixture for modes. I decided to use the [model_mommy](http://model-mommy.readthedocs.io/en/latest/) library so I could generate models easily without having detailed fixtures for everything.

### CODE REVIEW

File: survey/tests.py [hunk 1]
```diff
@@ -19,4 +21,44 @@ def test_create_mode(self):
...
+    def setUp(self):
+        self.survey_recipe = Recipe(models.Commutersurvey)
+        self.all_modes = models.Mode.objects.all()
+        self.leg_recipe = Recipe(
+            models.Leg,
+            checkin = self.survey_recipe.make(),
+            mode = cycle(self.all_modes)
+        )
+
+    def decimal_place_index(self, number):
```
Comment (pwnage101) [inline survey/tests.py]: if this function represents the number of decimal places in the given number, can you rename it to "decimal_places" ?

Comment (thecristen) [inline survey/tests.py]: Done

**Comment by pwnage101** [comment]
Sorry, was about to give LGTM but I was just thinking about adding a comment explaining why things are being rounded to 3 places.  Something like "round stored values to 3 decimal places for better human readability of the data dumps" at the top would suffice.  Other than that, LGTM.

**Comment by thecristen** [comment]
Done!

**Pull request merged**